### PR TITLE
return index of textbox

### PR DESCRIPTION
**Pull request by slootsky**
This allows developer to receive the index of the textbox which was just created.

Usage:
`dateBox=matrixportal.add_text(blahblahblah)
timeBox=matrixportal.add_text(blahblahblah)
...various code...

matrixportal.set_text('the date',dateBox)
matrixportal.set_text('the time',timeBox)
`

### CODE REVIEW

File: adafruit_matrixportal/network.py [hunk 1]
```diff
@@ -46,7 +46,7 @@
     )
     raise
 
-__version__ = "0.0.0-auto.0"
+__version__ = "1.9.4"
```
Comment (makermelissa) [inline adafruit_matrixportal/network.py]: Please leave the version at "0.0.0-auto.0" or the automatic versioning won't work.

File: adafruit_matrixportal/network.py [hunk 2]
```diff
@@ -295,15 +295,24 @@ def connect(self):
...
+                        secrets["ssid"] == "CHANGE ME"
+                        or secrets["password"] == "CHANGE ME"
+                    ):
+                        change_me = "\n" + "*" * 45
+                        change_me += "\nPlease update the 'secrets.py' file on your\n"
+                        change_me += "CIRCUITPY drive to include your local WiFi\n"
+                        change_me += "access point SSID name in 'ssid' and SSID\n"
+                        change_me += "password in 'password'. Then save to reload!\n"
+                        change_me += "*" * 45
+                        raise OSError(change_me)
```
Comment (makermelissa) [inline adafruit_matrixportal/network.py]: What is the purpose of the change for the code to check if a user has set their SSID/password yet?

**Review by makermelissa — CHANGES_REQUESTED** [review]
Please change the version back. Also, describe why you made the changes to the secrets check. Was this an accidental inclusion? That doesn't appear to be in the PR description.

File: adafruit_matrixportal/network.py (hunk 2 above)
Comment (slootsky) [reply adafruit_matrixportal/network.py]: yes, accidental inclusion.  Sorry, I'll fix that.

File: adafruit_matrixportal/network.py (hunk 1 above)
Comment (slootsky) [reply adafruit_matrixportal/network.py]: Will do.

File: adafruit_matrixportal/network.py (hunk 2 above)
Comment (slootsky) [reply adafruit_matrixportal/network.py]: [this is part of another change I have on my side to allow a list of SSID and a list of passwords, allowing the device to connect to a different wifi, depending on which is available]

**Review by makermelissa — APPROVED** [review]
Thanks, looks good.
